Library/Formula/ansible-cmdb.rb

```diff
@@ -16,11 +16,13 @@ class AnsibleCmdb < Formula
 
   def install
     bin.mkpath
+    man1.mkpath
     inreplace "Makefile" do |s|
       s.gsub! "/usr/local/lib/${PROG}", prefix
       s.gsub! "/usr/local/bin", bin
+      s.gsub! "/usr/local/share/man/man1/", man1 + 'ansible-cmdb.1'
+      s.gsub! "mandb", "#mandb"
```

Comment:
Regarding `mandb`, I created this upstream pull request: https://github.com/fboender/ansible-cmdb/pull/50